src/components/enterprise-user-subsidy/UserSubsidyAlerts.jsx

```diff
@@ -0,0 +1,31 @@
+import React, { useContext } from 'react';
+import { AppContext } from '@edx/frontend-platform/react';
+import { UserSubsidyContext } from '.';
+import OffersAlert from './OffersAlert';
+import SubscriptionSubsidy from './SubscriptionSubsidy';
+
+const UserSubsidyAlerts = () => {
+  const { enterpriseConfig, subscriptionPlan } = useContext(AppContext);
+  const { subscriptionLicense, offers } = useContext(UserSubsidyContext);
+
+  return (
+    <>
+      {offers && <OffersAlert offers={offers} />}
+      {/**
+       * SubscriptionSubsidy takes care of redirecting the user to `/${enterpriseConfig.slug}`
+       * if their organization has a subscription plan but they don't have appropriate access
+       * to a license (i.e., status="activated"). it also handles the case where the organization
+       * has an active subscription plan but the current date is not between the plan's start and
+       * expiration dates. The component also handles rendering warning/error status alerts.
+       */}
+      <SubscriptionSubsidy
+        enterpriseConfig={enterpriseConfig}
+        plan={subscriptionPlan}
+        license={subscriptionLicense}
+      />
```

Comment:
I'm preferring to leave the display components as simple as possible, and leaving the data handling to the parent component. It separates concerns and allows me to very easily swap out the display component without worrying about the data.

Reply:
I also don't consider passing props down one level to really be "prop drilling", and it simplifies the code by having us use AppContext and UserSubsidyContext in only one place instead of in both, so any changes only need to be made in the parent component.